### PR TITLE
Android unit tests for string resources

### PR DESCRIPTION
**Pull request by Tetraquark**
#25 Unit tests of string resources for Android with CI workflow action.

### CODE REVIEW

File: .github/workflows/ui-tests-android-sample.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+name: Unit tests of Android Sample
+
+on:
+  push:
+    branches:
+      - develop
```
Comment (Alex009) [inline .github/workflows/ui-tests-android-sample.yml]: change to:
```
on:
  pull_request:
    branches:
    - master
    - develop
```
to autocheck all requests without merge to develop

File: buildSrc/src/main/kotlin/Versions.kt [hunk 2]
```diff
@@ -23,6 +23,10 @@ object Versions {
     object Libs {
         object Android {
             const val appCompat = "1.1.0"
+
+            const val espressoCore = "3.2.0"
+            const val testRunner = "1.2.0"
+            const val testExtJunit = "1.1.1"
```
Comment (Alex009) [inline buildSrc/src/main/kotlin/Versions.kt]: move it to 
```
object AndroidTest {
    const val espressoCore = "3.2.0"
    const val testRunner = "1.2.0"
    const val testExtJunit = "1.1.1"
}
```
just like in Deps

File: sample/android-app/src/androidTest/java/com/icerockdev/StringResourcesTest.kt [hunk 3]
```diff
@@ -0,0 +1,46 @@
+package com.icerockdev
```
Comment (Alex009) [inline sample/android-app/src/androidTest/java/com/icerockdev/StringResourcesTest.kt]: add copyright here

File: sample/android-app/src/androidTest/java/com/icerockdev/StringResourcesTestEn.kt [hunk 4]
```diff
@@ -0,0 +1,20 @@
+package com.icerockdev
```
Comment (Alex009) [inline sample/android-app/src/androidTest/java/com/icerockdev/StringResourcesTestEn.kt]: add copyright here

File: sample/android-app/src/androidTest/java/com/icerockdev/StringResourcesTestRu.kt [hunk 5]
```diff
@@ -0,0 +1,20 @@
+package com.icerockdev
```
Comment (Alex009) [inline sample/android-app/src/androidTest/java/com/icerockdev/StringResourcesTestRu.kt]: add copyright here

File: sample/android-app/src/main/java/com/icerockdev/LocaleHandler.kt [hunk 6]
```diff
@@ -0,0 +1,30 @@
+package com.icerockdev
```
Comment (Alex009) [inline sample/android-app/src/main/java/com/icerockdev/LocaleHandler.kt]: add copyright here

File: sample/android-app/src/main/java/com/icerockdev/MainActivity.kt [hunk 7]
```diff
@@ -1,5 +1,6 @@
 package com.icerockdev
```
Comment (Alex009) [inline sample/android-app/src/main/java/com/icerockdev/MainActivity.kt]: add copyright here

File: .github/workflows/ui-tests-android-sample.yml (hunk 1 above)
Comment (Tetraquark) [reply .github/workflows/ui-tests-android-sample.yml]: Fixed.

File: buildSrc/src/main/kotlin/Versions.kt (hunk 2 above)
Comment (Tetraquark) [reply buildSrc/src/main/kotlin/Versions.kt]: Moved to AdroidTest.

File: sample/android-app/src/androidTest/java/com/icerockdev/StringResourcesTest.kt (hunk 3 above)
Comment (Tetraquark) [reply sample/android-app/src/androidTest/java/com/icerockdev/StringResourcesTest.kt]: Done.

File: sample/android-app/src/androidTest/java/com/icerockdev/StringResourcesTestRu.kt (hunk 5 above)
Comment (Tetraquark) [reply sample/android-app/src/androidTest/java/com/icerockdev/StringResourcesTestRu.kt]: Done.

File: sample/android-app/src/main/java/com/icerockdev/LocaleHandler.kt (hunk 6 above)
Comment (Tetraquark) [reply sample/android-app/src/main/java/com/icerockdev/LocaleHandler.kt]: Done.

File: sample/android-app/src/main/java/com/icerockdev/MainActivity.kt (hunk 7 above)
Comment (Tetraquark) [reply sample/android-app/src/main/java/com/icerockdev/MainActivity.kt]: Done.

**Pull request merged**